docs/350_REGISTRY.md

```diff
@@ -0,0 +1,55 @@
+# What is a registry?
+
+A `Registry` should be responsible for contract specific behavior and consolidate
```

Comment:
do we want to call this registry? I feel like it makes it a little confusing since it doesn't seem to be actually representing the Registry contract.

Reply:
I'm open to another name. I agree with the confusion and it's just supposed to represent higher level ideas that one would want the contract to do. Here, the idea of checking the performability of an upkeep continues to change at the contract level, but the concept / output is still the same.